### PR TITLE
chore: Move playground into apps folder

### PR DESCRIPTION
**Pull request by mdodgelooker**
Create `/apps` folder and move `/playground` into it. Next up, `/filter-demo` will go in there.

### CODE REVIEW

File: packages/components/README.md [hunk 1]
```diff
@@ -26,12 +26,6 @@ yarn: `yarn add --dev @types/lodash @types/react @types/react-dom @types/styled-
 
 ## Contributor Tips
 
-### Watching Build Changes
-
-Working on the components package? This package includes a `develop` script to make is easy to watch the code for changes and have it automatically recompiled.
-
-- **yarn workspace @looker/components develop** builds the component library with a `--watch` flag. Useful when actively developing a component and you wish to see the changes appear in either the [style guide](./packages/www) or [react playground](./packages/playground)
-
```
Comment (mdodgelooker) [inline packages/components/README.md:34]: This is out of date so I removed it instead of fixing the path to playground.

File: yarn.lock [hunk 2]
```diff
@@ -2385,103 +2385,6 @@
     npmlog "^4.1.2"
     write-file-atomic "^3.0.3"
 
-"@looker/components-providers@*", "@looker/components-providers@^1.3.1":
```
Comment (mdodgelooker) [inline yarn.lock]: I think this diff may be b/c `main`'s `yarn.lock` is out of date?

**Review by lukelooker — DISMISSED** [review]
Change looks good and `playground` boots when on the branch :)

**Review by lukelooker — APPROVED** [review]
Re-approving (had merge conflict)

**Pull request merged**